skimage/transform/steerable.py

```diff
@@ -228,28 +257,33 @@ def recon_pyr_level(self, coeff, log_rad, Xrcos, Yrcos, angle):
             nlog_rad = log_rad[lostart[0]:loend[0], lostart[1]:loend[1]]
             nangle = angle[lostart[0]:loend[0], lostart[1]:loend[1]]
             YIrcos = np.sqrt(np.abs(1 - Yrcos * Yrcos))
-            lomask = point_op(nlog_rad, YIrcos, Xrcos)
+            lomask = _point_op(nlog_rad, YIrcos, Xrcos)
 
-            nresdft = self.recon_pyr_level(
+            nresdft = self._recon_pyr_level(
                 coeff[1:], nlog_rad, Xrcos, Yrcos, nangle)
 
-            res = np.fft.fftshift(np.fft.fft2(nresdft))
-
             resdft = np.zeros(dims, 'complex')
             resdft[lostart[0]:loend[0], lostart[1]:loend[1]] = nresdft * lomask
 
             return resdft + orientdft
 
     def recon_scf_pyramid(self, coeff):
-        """
+        """ Reconstruc the image from its Steerable decomposition
```

Comment:
typo: missing a t on Reconstruct.  Also, please add a period to the end of the sentence.